Eatery/Views/Eateries/MealItemTableViewCell.swift

```diff
@@ -21,12 +29,59 @@ class MealItemTableViewCell: UITableViewCell {
         addSubview(nameLabel)
         nameLabel.translatesAutoresizingMaskIntoConstraints = false
         nameLabel.snp.makeConstraints { make in
-            make.edges.equalTo(self).inset(UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16))
+            make.edges.equalTo(self).inset(UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16))
```

Comment:
Style: use `make.edges.equalToSuperview()`. This is what's been done everywhere else